src/test/scala/com/cognite/sdk/scala/v1/ClientTest.scala

```diff
@@ -104,7 +104,8 @@ class ClientTest extends SdkTestSpec {
        |    "user": "tom@example.com",
        |    "loggedIn": true,
        |    "project": "${loginStatus.project}",
-       |    "projectId": ${loginStatus.projectId}
+       |    "projectId": "${loginStatus.projectId}",
```

Comment:
We should also add tests that parse a response such as it is returned from `login/status` when using an invalid API key:
```javascript
{
  "data": {
    "user": "",
    "loggedIn": false,
    "project": "",
    "projectId": -1
  }
}
```